css/style.css

```diff
@@ -0,0 +1,871 @@
+}
+
+.amateurs {
+	background: url(../img/amateurs.png) no-repeat center top;
+}
+
+.everyone {
+	background: url(../img/everyone.png) no-repeat center top;
+}
+
+#more-information {
+	text-align: center;
+	background: #f5f5f5;
+}
+
+.more-information-wrap {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.resp-item-footer {
+	padding: 10px;
+}
+
+.resp-item-footer h5 {
+	text-transform: uppercase;
+}
+
+.resp-item-footer h5.title{
+	margin-bottom: 50px;
+}
+
+.btn-info {
+	display: inline-block;
+	background: #50c3af; 
+	margin: 30px 20px;
+	padding: 10px 10px;
+	text-transform: uppercase;
+	font: 14px Arial, sans-serif;
+	color: #f5f5f5;
+	-webkit-border-radius: 20px;
+	-moz-border-radius: 20px;
+	-o-border-radius: 20px;
+	border-radius: 20px;
+	cursor: pointer;
+}
+
+.btn-info:nth-last-child(1),
+.btn-info:nth-last-child(2) {
+	margin-bottom: 0;
+}
+
+.btn-info:hover {
+	background: #34ab96;
+	color: #fff;
+}
+
+.supported-template{
+	background: #eeeeef;
+}
+
+.sup-item {
+	display: flex;
+	align-items: center;
+}
+
+.sup-item h5 {
+	font: bold 18px Arial, sans-Serif;
+	text-transform: uppercase;
+	padding-left: 15px;
+	color: #68767a;
+}
+
+.suported-item p {
+	font: 14px Arial, sans-Serif;
+	color: #787b80;
+}
+
+.btn-sup {
+	display: inline-block;
+	margin-top: 20px;
+	padding: 10px 10px;
+	color: #f5f5f5;
+	background: #d2d5da;
+	-webkit-border-radius: 20px;
+	-moz-border-radius: 20px;
+	-o-border-radius: 20px;
+	border-radius: 20px;
+	cursor: pointer;
+}
+
+.btn-sup:hover {
+	background: #878c94;
+}
+
+/*---*/
+
+#layered-psd {
+	background: #f5f5f5;
+}
+
+.layered-psd-wrap {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.layered-psd-item {
+	display: block;
+}
+
+.layered-item-footer h5 {
+	text-transform: uppercase;
+}
+
+.layered-item-footer h5.title{
+	margin-bottom: 50px;
+}
+
+.layered-item-footer {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	align-items: flex-start;
+}
+
+.link-info a{
+	display: block;
+	color: #50c3af; 
+	margin: 30px 0;
+	padding: 10px 0;
+	text-transform: uppercase;
+	font: 14px Arial, sans-serif;
+	cursor: pointer;
+}
+
+.link-info:nth-last-child(1),
+.link-info:nth-last-child(2) {
+	margin-bottom: 0;
+}
+
+.link-info a:hover {
+	color: #34ab96;
+}
+
+/*---*/
+
+#send-information {
+	background: #eeeeef;
+}
+
+.send-information-wrap {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	align-items: center;
+}
+
+.read-more-item {
+	display: block;
+	width: 40%;
+}
+
+.send-item {
+	display: flex;
+	align-items: center;
+}
+
+.send-item h5 {
+	font: bold 18px Arial, sans-serif;
+	text-transform: uppercase;
+	padding-left: 15px;
+	color: #384048;
+}
+
+.read-item-footer {
+	padding: 10px;
+}
+
+.read-item-footer span{
+	color: #787b80;
+	font: 14px Arial, sans-serif;
+	line-height: 25px;
+}
+
+.btn-read {
```

Comment:
`.btn-info`, `.btn-sup`, `.btn-read` have many common rules. Think how you can reorganize code to prevent repeating yourself?